### ISSUE TITLE
Mobile-O-0.5B pre_train.sh Does Not Match the Paper

### ISSUE THREAD

**Issue by mysticalwing** [Amshaker/Mobile-O#5]
Hi authors, thank you for your great work. I am trying to reproduce your results, but I noticed that some hyperparameters in pre_train.sh do not match those reported in the paper. For example, the script sets weight_decay to 0, while the paper reports 0.01; the script uses a learning rate of 5e-5, while the paper states 2e-4. Could you clarify which settings I should follow for reproduction? 
Thank you in advance for your help, and I look forward to your reply.

**Comment by Amshaker** [comment]
Hi @mysticalwing ,

Thank you for your interest in our work and for catching this!

I’ve rechecked the original training script, and it aligns with the reported numbers in the paper. I have now updated both pretrain.sh (for 0.5B and 1.5B) with the correct hyperparameters. Specifically, the updated values are:

```
--learning_rate 2e-4 \
--weight_decay 0.01 \
--warmup_ratio 0.02 \
--lr_scheduler_kwargs '{"min_lr": 2e-6}' \
```

Thank you!

**Comment by mysticalwing** [comment]
Hi @Amshaker 

Thanks for the clarification and for updating the scripts. That clears things up a lot!

I’ll follow the updated pretrain.sh settings for reproduction.

Thanks again!
